clientbrownfield/subcommands.go

```diff
@@ -11,74 +11,72 @@ func SubCommands() []cli.Command {
 	return []cli.Command{
 		{
 			Name:        "cloud-accounts",
-			Usage:       "Provides information about brownfield cloud accounts. Allows querying cloud accounts to discover (and list) candidate resources from IMCO.",
+			Usage:       "Provides information about brownfield cloud accounts. Allows querying cloud accounts to import candidate resources from IMCO.",
 			Subcommands: append(cloud_accounts.SubCommands()),
 		},
 		{
-			Name:   "import-server",
-			Usage:  "Import server import candidate, given its id",
-			Action: cmd.ImportCandidateServerImport,
+			Name:   "import-servers",
+			Usage:  "Import servers candidates, by given cloud account id",
+			Action: cmd.ImportCandidateServers,
 			Flags: []cli.Flag{
 				cli.StringFlag{
 					Name:  "id",
-					Usage: "Import candidate server Id",
-				},
-				cli.StringFlag{
-					Name:  "ssh-profile-id",
-					Usage: "Identifier of the ssh profile which the server shall use",
-				},
-				cli.StringFlag{
-					Name:  "ssh-profile-ids",
-					Usage: "A list of comma separated ssh profiles ids",
-				},
-				cli.StringFlag{
-					Name:  "labels",
-					Usage: "A list of comma separated label as a query filter",
+					Usage: "Cloud account Id",
 				},
 			},
 		},
 		{
-			Name:   "import-vpc",
-			Usage:  "Import VPC import candidate, given its id",
-			Action: cmd.ImportCandidateVPCImport,
+			Name:   "import-vpcs",
+			Usage:  "Import VPCs candidates, by given cloud account id",
+			Action: cmd.ImportCandidateVPCs,
 			Flags: []cli.Flag{
 				cli.StringFlag{
 					Name:  "id",
-					Usage: "Import candidate VPC Id",
-				},
-				cli.StringFlag{
-					Name:  "labels",
-					Usage: "A list of comma separated label as a query filter",
+					Usage: "Cloud account Id",
 				},
 			},
 		},
 		{
-			Name:   "import-floating-ip",
-			Usage:  "Import Floating IP import candidate, given its id",
-			Action: cmd.ImportCandidateFloatingIPImport,
+			Name:   "import-floating-ips",
+			Usage:  "Import Floating IPs candidates, by given cloud account id",
+			Action: cmd.ImportCandidateFloatingIPs,
 			Flags: []cli.Flag{
 				cli.StringFlag{
 					Name:  "id",
-					Usage: "Import candidate floating IP Id",
+					Usage: "Cloud account Id",
 				},
+			},
+		},
+		{
+			Name:   "import-volumes",
+			Usage:  "Import volumes candidates, by given cloud account id",
+			Action: cmd.ImportCandidateVolumes,
+			Flags: []cli.Flag{
 				cli.StringFlag{
-					Name:  "labels",
-					Usage: "A list of comma separated label as a query filter",
+					Name:  "id",
+					Usage: "Cloud account Id",
 				},
 			},
 		},
 		{
-			Name:   "import-volume",
-			Usage:  "Import volume import candidate, given its id",
-			Action: cmd.ImportCandidateVolumeImport,
+			Name:   "import-kb-clusters",
```

Comment:
- If your are going to shorten the Kubernetes word, use the widespread abbreviation `k8s`
- We should not add a kubernetes cluster import command until the CLI manages kubernetes clusters

Reply:
understood, kubernetes command is removed from current context; hiding command until kubernetes management is implemented.